_includes/commands/advtable-cmds.md

```diff
@@ -0,0 +1,27 @@
+
+| Command                  | Description                                                                                     |
+| ------------------------ | ----------------------------------------------------------------------------------------------- |
+| mceAdvancedTableSort     | Opens the Advanced Table Sort Dialog for the current selection or cursor location.              |
+| mceSortTableAdvanced     | Performs an Advanced Table Sort. See below for details.                                         |
+| mceSortTableByColumnAsc  | Sorts the current table ascending by column based on the current cursor position or selection.  |
+| mceSortTableByColumnDesc | Sorts the current table descending by column based on the current cursor position or selection. |
+| mceSortTableByRowDesc    | Sorts the current table ascending by row based on the current cursor position or selection.     |
```

Comment:
These two are back to front, as the name doesn't match the description.

Reply:
Fixed